### PR TITLE
Showdown - Added support for setFlavor() - #13302

### PR DESCRIPTION
**Pull request by liammann**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [ ] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/showdownjs/showdown
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.

### CODE REVIEW

File: showdown/index.d.ts [hunk 1]
```diff
@@ -255,6 +255,14 @@ declare namespace Showdown {
          * @param extensions
          */
         removeExtension(extensions: ShowdownExtension[] | ShowdownExtension): void;
+        
+        /**
+         * Setting a "local" flavor affects all instances of showdown
```
Comment (tan9) [inline showdown/index.d.ts]: Is the instance function take effects on other instances?

Comment (liammann) [inline showdown/index.d.ts]: Ops, yeah the comment should say, for only the current Converter instance

**Comment by minestarks** [comment]
@tan9 look good?

**Comment by tan9** [comment]
@minestarks Yes, I think this PR can be merged.

**Pull request merged**